package.json

```diff
@@ -60,6 +60,7 @@
     "redux-thunk": "3.1.0",
     "throttle-debounce": "5.0.0",
     "xterm": "5.1.0",
-    "xterm-addon-canvas": "0.5.0"
+    "xterm-addon-canvas": "0.5.0",
+    "@novnc/novnc": "1.3.0"
```

Comment:
This *downgrades* novnc from 1.5.0 to 1.3.0, see [diff](https://github.com/cockpit-project/node-cache/compare/a0d35ea1084df4dcf9f26d24ebfdaf531cb95e63..19ad5e70c0ca5784a42e347e4b2d0437aadb00f6). I don't think that's what we want? 1.5.0 itself worked fine, only later 1.5.x broke?

Reply:
Ah, of course, sorry!

Reply:
But the [total diff](https://github.com/cockpit-project/node-cache/compare/5f9629f2a21d6998777379c7cac23d7e6e37feee..19ad5e70c0ca5784a42e347e4b2d0437aadb00f6#diff-ef2cfb6479ad59efdd278d31be29bb29141f08dea8194979a4078675366f6de5L400) still shows it's downgrading from 1.4.0 to 1.3.0. Surely we should at least keep that?

Reply:
Yes, lets now downgrade but stay at 1.4.0. Thanks for spotting!